core/runtime/common/uncompress_code_if_needed.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime/common/uncompress_code_if_needed.hpp"
+
+#include <zstd.h>
+
+#include "log/logger.hpp"
+
+namespace kagome::runtime {
+
+  // @see
+  // https://github.com/paritytech/substrate/blob/polkadot-v0.9.8/primitives/maybe-compressed-blob/src/lib.rs#L28
+  constexpr uint8_t kZstdPrefix[8] = {
+      0x52, 0xBC, 0x53, 0x76, 0x46, 0xDB, 0x8E, 0x05};
+  // @see
+  // https://github.com/paritytech/substrate/blob/polkadot-v0.9.8/primitives/maybe-compressed-blob/src/lib.rs#L35
+  constexpr size_t kCodeBlobBombLimit = 50 * 1024 * 1024;
+
+  void uncompressCodeIfNeeded(const common::Buffer &buf, common::Buffer &res) {
+    if (buf.size() > 8
```

Comment:
I suggest using named constants instead of magic numbers.

Reply:
`std::size(kZstdPrefix)`